enterprise/validators.py

```diff
@@ -0,0 +1,24 @@
+"""
+Database models field validators.
+"""
+import os
+from django.core.exceptions import ValidationError
+from django.utils.translation import ugettext_lazy as _
+
+
+def validate_image_extension(value):
+    """
+    Validate that a particular image extension.
+    """
+    ext = os.path.splitext(value.name)[1]  # [0] returns path+filename
+    valid_extensions = ['.png', ]
+    if not ext.lower() in valid_extensions:
+        raise ValidationError(u'Unsupported file extension.')
+
+
+def validate_image_size(image):
+    """
+    Validate that a particular image size.
+    """
+    if not image.size < (250 * 1024):
```

Comment:
@asadiqbal08 nit: extract size into setting in `app.py`?

Bonus-level 1: make constant an overridable setting (and put it into app config in app.py)
Bonus-level 2: using ninja skills, make `logo` field help text use that setting.

Bonus levels are **really** optional.